mpp/include/mpp_util.inc

```diff
@@ -1348,8 +1348,7 @@ end function rarray_to_char
     integer, dimension(2) :: get_ascii_file_num_lines_and_length !< number of lines (1) and
                                                                  !! max line length (2)
     integer :: num_lines, max_length
-    integer, parameter :: LENGTH=1024
-    character(len=LENGTH) :: str_tmp
+    character(len=FMS_FILE_LEN) :: str_tmp
```

Comment:
This is not a filename, but the length of a string being read from an ascii file for conversion to a character variable.  Using FMS_FILE_LEN would now impose a limit of 255 from the original 1024.

Reply:
Fixed with f9cac03